### PR TITLE
Enable netinfo in add metadata processors

### PR DESCRIPTION
**Pull request by jsoriano**
## What does this PR do?

Enable netinfo in add_host_metadata and add_observer_metadata processors by default.

## Why is it important?

This network information metadata is used in observability solutions.

This change shouldn't break backwards compatibility or produce any conflict as the added fields are in ECS. 

## Checklist

- [x] My code follows the style guidelines of this project
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] I have made corresponding change to the default configuration files
- [x] I have added tests that prove my fix is effective or that my feature works

## How to test this PR locally

* Run a beat with the `add_host_metadata` processor without any options.
* Check that all events contain `host.ip` and `host.mac` fields.

## Related issues

- Superseds elastic/beats#15347

### CODE REVIEW

File: libbeat/processors/add_host_metadata/docs/add_host_metadata.asciidoc [hunk 1]
```diff
@@ -5,7 +5,7 @@
 -------------------------------------------------------------------------------
 processors:
 - add_host_metadata:
-    netinfo.enabled: false
+    netinfo.enabled: true
```
Comment (kaiyan-sheng) [inline libbeat/processors/add_host_metadata/docs/add_host_metadata.asciidoc]: Do we still need this with func defaultConfig() has `NetInfoEnabled: true`?

Comment (jsoriano) [reply libbeat/processors/add_host_metadata/docs/add_host_metadata.asciidoc]: Not really needed, but it was also not needed before, I can remove it. I don't have an opinion on this, so as you prefer.

Comment (kaiyan-sheng) [reply libbeat/processors/add_host_metadata/docs/add_host_metadata.asciidoc]: Cool, I just want to make sure it's consistent with the rest of the files in Beats. How about removing it?

Comment (jsoriano) [reply libbeat/processors/add_host_metadata/docs/add_host_metadata.asciidoc]: Removed :+1:

**Comment by jsoriano** [comment]
> ``.

Yep, I had similar thoughts https://github.com/elastic/beats/pull/15347#issuecomment-579685416

**Comment by jsoriano** [comment]
I have made the same change in `add_observer_metadata` for consistency. I would find weird to see one adding this data by default while the other doesn't. Opinions?

**Comment by jsoriano** [comment]
Failing test is also failing in master, it seems to be related to a change in ES 8.0 snapshots.

File: libbeat/processors/add_host_metadata/docs/add_host_metadata.asciidoc (hunk 1 above)
Comment (kaiyan-sheng) [reply libbeat/processors/add_host_metadata/docs/add_host_metadata.asciidoc]: Thanks!

**Pull request merged**